api/views.py

```diff
@@ -263,3 +266,15 @@ class GetGoogleApiSecret(APIView):
     def get(self, request, format=None):
         content = {'secret_key': settings.GOOGLE_API_SECRET_KEY}
         return Response(content)
+
+
+token_f = openapi.Parameter('token', openapi.IN_QUERY, description="Check if Token is valid", type=openapi.TYPE_STRING)
+
+
+@swagger_auto_schema(method='get', manual_parameters=[token_f])
+@api_view(['GET'])
+def ValidateToken(request):
+    token = request.query_params['token']
+    queryset = Token.objects.all()
+    isValid: bool = token in [x.key for x in queryset]
```

Comment:
While there should not be many tokens, it is much better to check if the token exists in the DB without iterating over them. Use queryset.filter() and queryset.exists().